docs/reference/cluster/nodes-info.asciidoc

```diff
@@ -186,20 +247,7 @@ The result will look similar to:
 // TESTRESPONSE[s/"plugins": \[[^\]]*\]/"plugins": $body.$_path/]
 // TESTRESPONSE[s/"modules": \[[^\]]*\]/"modules": $body.$_path/]
 
```

Comment:
Not required, but you may want to consider adding a heading here to better separate the two examples.